src/users/editUser.ts

```diff
@@ -15,28 +15,48 @@ export const handler = new LambdaBuilder(updateRequest)
 export async function updateRequest(
     event: APIGatewayProxyEvent
 ): Promise<APIResponse> {
+    if (!event.pathParameters) {
+        throw new Error('Event parameters is missing');
+    }
+
+    if (!event.body) {
+        throw new Error('Event body missing');
+    }
+
+    const updatePersonData = JSON.parse(event.body) as UpdatePerson;
+
     await updateUser(
         event.pathParameters.id as string,
         JSON.parse(event.body) as UpdatePerson
     );
     return new SuccessResponse({
-        message: `user with id : ${event.pathParameters.id} updated`,
+        message: `User with id : ${event.pathParameters.id} updated successfully`,
+        updateUser: updatePersonData,
```

Comment:
this object could just be named `user` since it will be easier for the frontend to modularize the code then